src/components/date/__spec__.js

```diff
@@ -381,6 +385,10 @@ describe('Date', () => {
       it('does not open the date picker', () => {
         expect(instance.openDatePicker).not.toHaveBeenCalled();
       });
+
+      it('sets the input as disabled', () => {
+        expect(instance._input.disabled).toEqual(true);
```

Comment:
Ah, but `input` elements have a `readonly` attribute too: https://developer.mozilla.org/en/docs/Web/HTML/Element/input#attr-readonly

So I think the `readOnly` prop should set the `readonly` attribute.